docs/src/rules/no-extra-parens.md

```diff
@@ -130,6 +131,24 @@ for (;(a = b););
 
 :::
 
+### ternaryOperandBinaryExpressions
+
+Examples of **correct** code for this rule with the `"all"` and `{ "ternaryOperandBinaryExpressions": false }` options:
+
+```js
+/* eslint no-extra-parens: ["error", "all", { "ternaryOperandBinaryExpressions": false }] */
+
+(a && b) ? foo : bar
+
+(a - b > a) ? foo : bar
+
+foo ? (bar || baz) : qux
+
+foo ? bar : (baz || qux)
+
+(a, b) ? (c, d) : (e, f)
```

Comment:
This example might be confusing because all these parentheses are required for precedence, so they're allowed regardless of this option. It might be better to remove it.